include/fi.h

```diff
@@ -209,8 +235,12 @@ int fi_rma_target_allowed(uint64_t caps);
         asm(".symver " #name "," #api "@@" DEFAULT_ABI)
 #else
 #  define symver(name, api, ver)
+#ifdef  HAVE_ALIAS_ATTRIBUTE
```

Comment:
So OS X compiler (xcode? or gcc? or clang? I am not familiar with OS X) supports neither alias or symbol versions?